### PR TITLE
AST-2792 - fix: Support for button preset outline (buttons 4-6) styles in editor

### PR DESCRIPTION
**Pull request by neerajmasai**
### Description

- The problem occurs with button presets outline styles, editor shows bg color even for outline design preset (buttons 4-5), which is not expected.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2792

### Screenshots

https://d.pr/v/exFxcf
### Types of changes

- bugfix - The problem occurs with button presets outline styles, editor shows bg color even for outline, which is not expected.

### How has this been tested?

- Global > Buttons > Button Presets
- Select a outline design from available button design options.
- Check in frontend & compare with editor.
- The buttons should look same in terms of design

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/core/class-astra-wp-editor-css.php [hunk 1]
```diff
@@ -343,7 +343,24 @@ public static function get_css() {
...
+		if ( 'button_04' === $btn_preset_style || 'button_05' === $btn_preset_style || 'button_06' === $btn_preset_style ) {
+
+			if ( empty( $btn_border_color ) ) {
+				$btn_border_color = $btn_bg_color;
+			}
+
+			if ( '' === astra_get_option( 'button-bg-color' ) && '' === astra_get_option( 'button-color' ) ) {
+				$btn_color = $theme_color;
+			} elseif ( '' === astra_get_option( 'button-color' ) ) {
+					$btn_color = $btn_bg_color;
```
Comment (imnavanath) [inline inc/core/class-astra-wp-editor-css.php:358]:
```suggestion
				$btn_color = $btn_bg_color;
```

File: inc/core/class-astra-wp-editor-css.php [hunk 2]
```diff
@@ -523,7 +540,7 @@ public static function get_css() {
 				'border-right-width'  => $theme_btn_right_border,
 				'border-left-width'   => $theme_btn_left_border,
 				'border-bottom-width' => $theme_btn_bottom_border,
-				'color'               => esc_attr( $btn_color ),
+				'color'               => empty( $btn_color ) ? esc_attr( $btn_color ) : esc_attr( $btn_color ),
```
Comment (imnavanath) [inline inc/core/class-astra-wp-editor-css.php:543]: @neerajmasai can you check this statement once?

Comment (neerajmasai) [reply inc/core/class-astra-wp-editor-css.php:543]: @imnavanath My bad, it is accidental, not intended. I will fix this.

File: inc/core/class-astra-wp-editor-css.php (hunk 1 above)
Comment (neerajmasai) [reply inc/core/class-astra-wp-editor-css.php]: @imnavanath Resolved.

File: inc/core/class-astra-wp-editor-css.php (hunk 2 above)
Comment (neerajmasai) [reply inc/core/class-astra-wp-editor-css.php]: @imnavanath Resolved.

**Review by imnavanath — APPROVED** [review]

**Review by yssambare12 — APPROVED** [review]

**Pull request merged**